src/components/ProjectDashboard/hooks/useProjectHeader.ts

```diff
@@ -36,7 +37,22 @@ export const useProjectHeader = (): ProjectHeaderData => {
   const subtitle = useMemo(() => {
```

Comment:
could also just do `const subtitle: { text: string, type: SubtitleType } | undefined` instead of casting below

Reply:
Yep fair point! I just thought this way was a bit cleaner, but also definitely valid